lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_kosovo_when_residing_in_uk_or_kosovo.govspeak.erb

```diff
@@ -34,7 +34,8 @@
 
     You normally have to pay fees for consular services in the local currency - these are shown in the list of [consular fees for <%= calculator.country_name_lowercase_prefix %>](/government/publications/<%= calculator.ceremony_country %>-consular-fees).
```

Comment:
This line of content seems to include an aspect of how to pay i.e. "in the local currency". Do you think that also needs to be extracted somehow?